package.json

```diff
@@ -85,7 +84,7 @@
     "ember-papaparse": "0.0.3",
     "ember-pikaday": "2.2.1",
     "ember-promise-helpers": "1.0.2",
-    "ember-resolver": "^3.0.0",
+    "ember-resolver": "^2.0.3",
```

Comment:
Should also pull resolver out of the greenkeeper->ignore section at the bottom of `package.json`.